src/wp-includes/class-wp-script-modules.php

```diff
@@ -331,19 +331,26 @@ function ( $dependency_script_modules, $id ) use ( $import_types ) {
 	 *
 	 * @since 6.5.0
 	 *
-	 * @param array $script_module The script module.
+	 * @param string $id The script module identifier.
 	 * @return string The script module src with a version if relevant.
 	 */
-	private function get_versioned_src( array $script_module ): string {
-		$args = array();
+	private function get_src( string $id ): string {
```

Comment:
I originally changed this to `get_versioned_src( $script_module, $id )` but it didn't make much sense given the former can be inferred from the latter.

The areas that calls this function could probably be optimized not to fetch the `$script_module` array and just the IDs, but I chose against doing that in this PR for simplicity.